packages/python-packages/apiview-copilot/src/mention/_open_parser_issue_workflow.py

```diff
@@ -0,0 +1,84 @@
+import json
+import prompty
+from src._github_manager import GithubManager
+from src._utils import get_prompt_path
+from ._base import MentionWorkflow
+
+
+class OpenParserIssueWorkflow(MentionWorkflow):
+    prompty_filename = "parse_conversation_to_github_issue.prompty"
+    summarize_prompt_file = "summarize_github_actions.prompty"
+    deduplication_prompt_file = "deduplicate_parser_issue.prompty"
+    
+    REPO_OWNER = "tjprescott"
+    REPO_NAME = "azure-sdk-tools"
+
+    def execute_plan(self, plan: dict):
+        """Execute the parser issue workflow"""
+        recent_issues = self._fetch_recent_parser_issues()
+        dedup_result = self._check_for_duplicate_issue(plan, recent_issues)
+        
+        if dedup_result["action"] == "no-op":
+            return {
+                "action": "no-op",
+                "existing_issue": dedup_result["existing"],
+            }
+        
+        issue = self._create_parser_issue(plan)
+        return {
+            "action": "created",
+            "issue": issue
+        }
+
+    def _fetch_recent_parser_issues(self):
+        """Fetch recent open parser issues from GitHub."""
+        client = GithubManager.get_instance()
+        issues = client.search_issues(
+            owner=self.REPO_OWNER,
+            repo=self.REPO_NAME,
+            query="label:parser is:issue state:open"
```

Comment:
We can't query on label because we cannot create new labels (even if it lets you--don't!). You could query for "APIView parser" or something. Possibly also azure-sdk-automation as the author, though that would ignore issues posted by humans (which I'd advise against).